packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -312,10 +391,10 @@ export const NetworkSelectionContainer = ({
                   secondPageTitle="Networks"
                   isLoading={false}
                   loadingMessage="Fetching Networks..."
-                  bottomRightCTAtext=""
+                  bottomRightCtaText=""
```

Comment:
Can we not pass the prop instead, if we want empty text?

Reply:
yes definitely, i must have missed this when i cleaned up